bookshelf/php.ini

```diff
@@ -0,0 +1 @@
+extension=mongodb.so
```

Comment:
Small downside might be increased memory consumption. This can be discussed on `php-docker` repo, although I would not enable mongodb by default.